kvdb/etcd/readwrite_bucket.go

```diff
@@ -155,10 +141,6 @@ func (b *readWriteBucket) CreateBucket(key []byte) (
 		return nil, walletdb.ErrBucketExists
 	}
 
-	if err := b.assertNoValue(key); err != nil {
```

Comment:
Just out of curiosity: What happens if you try to create a bucket in `bbolt` where a value already exists? Do you get a hard error?

Reply:
Yes with `bbolt` you get `ErrIncompatibleValue`. Here's the relevant code from `bbolt`:

```
	// Return an error if there is an existing key.
	if bytes.Equal(key, k) {
		if (flags & bucketLeafFlag) != 0 {
			return nil, ErrBucketExists
		}
		return nil, ErrIncompatibleValue
	}
```